hledger-lib/Hledger/Data/Journal.hs

```diff
@@ -293,9 +293,9 @@ journalEquityAccountQuery _ = Acct "^equity(:|$)"
 -- | A query for Cash (-equivalent) accounts in this journal (ie,
 -- accounts which appear on the cashflow statement.)  This is currently
 -- hard-coded to be all the Asset accounts except for those containing the
--- case-insensitive regex @(receivable|A/R)@.
+-- case-insensitive regex @(receivable|A/R|fixed)@.
```

Comment:
Perhaps :fixed would be better ? So as not to match assets:unfixed, assets:non-fixed etc.